src/components/verification-grid/verification-grid.ts

```diff
@@ -87,6 +96,32 @@ export class VerificationGridComponent extends AbstractComponent(LitElement) {
   public static readonly decisionMadeEventName = "decision-made" as const;
   private static readonly loadedEventName = "loaded" as const;
 
+  @provide({ context: verificationGridContext })
+  @property({ attribute: false })
+  public settings: VerificationGridSettings = {
+    axes: signal(true),
+    indicator: signal(true),
+    mediaControls: signal(true),
+  };
+
+  /** Whether to show the media controls component in each grid tile */
+  @property({ attribute: "media-controls", converter: booleanConverter })
+  public set mediaControls(value: boolean) {
+    this.settings.mediaControls.value = value;
+  }
+
+  /** Whether to show the axes component in each grid tile */
+  @property({ attribute: "axes", converter: booleanConverter })
+  public set axes(value: boolean) {
+    this.settings.axes.value = value;
+  }
+
+  /** Whether to show the indicator component in each grid tile */
+  @property({ attribute: "indicator", converter: booleanConverter })
+  public set indicator(value: boolean) {
+    this.settings.indicator.value = value;
+  }
+
```

Comment:
question: are getters still defined?

Reply:
No, which is really cool

![image](https://github.com/user-attachments/assets/d68c4c4c-4457-46c4-b8de-7242a55aeb2d)

Reply:
I'd say more confusing than not. If this is part of the public api then we probably want the getters created

Reply:
Agreed.

This property has been removed as per a previous comment